pkgs/development/python-modules/trubar/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  gitUpdater,
+  pytestCheckHook,
+  libcst,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "trubar";
+  version = "0.3.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "janezd";
+    repo = "trubar";
+    rev = "refs/tags/${version}";
+    hash = "sha256-FjczKGqRILTuhcTJqufQoJpyENuEkIIBdZkDoEkC3+g=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    libcst
+    pyyaml
+  ];
+
+  pythonImportsCheck = [ "trubar" ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  passthru.updateScript = gitUpdater { };
```

Comment:
Why not `nix-update-script` ?

Reply:
Simpler, and the actual code is in nixpkgs already. BTW there is a generic updater that you can setup by just giving a command that list the versions available

Reply:
Ok, I was not aware of that. So basically it is (or should be) recommended over `nix-update-script` for all packages using `fetchFromGit*` ?

Reply:
Anything using git should work.

If they use some kind of tag prefix such as `v` or `sscg-` then you pass it using the rev-prefix parameter and that's it.

If you want always the latest commit then you use `unstableGitUpdater`, and there is `genericUpdater` when the source is not git, you basically pass a script to list versions then it takes the latest one by comparison then change the hashes automatically. This version list is a bit clunky because the system expects you to pass a expression that takes package information as an argument, I did an example of that in #354277 tho.

Reply:
Ok good to know ! Thank you for the explanation :)

Reply:
Actually, I think that specifying the updater is useless for packages under `python3Packages` as they all inherit an updater (`nix-update-script` I think) by default.

Hence, I would actually get rid of those.
Maybe @mweinelt can confirm.